README.md

```diff
@@ -8,13 +8,15 @@ The Student Robotics public website.
 
 1. [Install Ruby 2.7][install-ruby]
 
-2. Install Bundler (1.x) and Rake
+2. Install [Node.js and npm][install-node]
```

Comment:
I think we should probably make it really clear that the NPM stuff _cannot_ be used to pull in JS frameworks or anything -- that it's a developer tool only.
(Maybe here and also a comment in the `package.json`)

A wild idea here might be to git ignore the `package.json` and have the Rakefile do `npm install cspell==$VERSION` (or however you spell that), so that it's really obvious that it's not a general purpose thing.

Reply:
> A wild idea here might be to git ignore the `package.json` and have the Rakefile do `npm install cspell==$VERSION` (or however you spell that), so that it's really obvious that it's not a general purpose thing.

For clarity: I'm not actually sure this is a good idea; very open to alternatives that make it similarly obvious.